spec/functional/honour_origin_cache_headers_spec.rb

```diff
@@ -1,9 +1,66 @@
 describe 'honour_origin_cache_headers' do
+
+  let(:a_date_in_the_future) { 'Thu, 01 Dec 2015 07:00:00 GMT' }
+  let(:a_date_in_the_future_plus_one) { 'Thu, 01 Dec 2015 07:01:00 GMT' }
```

Comment:
I did try making it relative but I got stuck trying to extract the right format string from Ruby's Time API. I had previously been trying to find the correct answer in the RFC documents (822, 2616 and 1123) but just got myself confused. However! Looking at the problem again, I see the method we need here is just Time#httpdate.

This should do the trick: 
let(:a_date_in_the_future) { (Time.now + 6*30*24*3600).httpdate }
let(:a_date_in_the_future_plus_one) { (Time.now + 6*30*24*3600 + 1).httpdate }